File: WebServices_lab/src/main/java/org/example/component/Row.java
package org.example.component;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Row implements Serializable {
    @JsonProperty("Values")
    public List<String> values = new ArrayList<>();

    @JsonCreator
    public Row(@JsonProperty("Values") List<String> values) {
        this.values = values;
    }

    public String getAt(int index){
        return values.get(index);
    }

    public void setAt(int index, String content){
        values.set(index,content);
    }
}

File: WebApplication_lab/Classes/Row.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication.Classes
{
    [Serializable]
    public class Row
    {
        public List<string> Values { get; } = new List<string>();

        public string GetAt(int index)
        {
            return Values[index];
        }

        public void SetAt(int index, string content)
        {
            Values[index] = content;
        }
    }
}
